tests/pa3-visible.test.ts

```diff
@@ -33,7 +33,6 @@ print(x)` , [`5`]);
     `
 class C(object):
   x : int = 123
-
```

Comment:
I think it would be better not to introduce diffs into this test file. (Was something broken that needed these newlines removed?)

Reply:
We mistakenly modified this file.Fixed now.